src/test-ids.ts

```diff
@@ -16,5 +18,9 @@ export const TestIds = {
   addWendlerLegDayButton: "add-wendler-leg-day-button",
   // PreferencesPage test IDs
   Preferences_SavePreferencesButton: "save-user-preferences-button",
-  // Add more test IDs here as needed
+  SelectRepsAMRAPToggle: "select-reps-amrap-toggle",
+  SelectRepsAdd: "select-reps-add-1",
+  PerceivedEffort: (effort: PerceivedEffort) => `perceived-effort-${effort}`,
```

Comment:
This is a relatively common pattern I've landed on, creating helper functions when it'd be really annoying to otherwise get unique test ids for like an enum value